c++/src/kj/compat/http.c++

```diff
@@ -1613,6 +1613,15 @@ private:
 template <char...>
 struct FastCaseCmp;
 
+// Disable tautological compare warning/error seen on gcc 10.  It triggers on patterns like
```

Comment:
An alternative solution that I like a little better, if it works: https://github.com/ecatmur/capnproto/commit/c38629e5a4b8ce9561b81cb23ea5fc39cbd93eb7

@ecatmur Were you planning to make a PR for that commit?

Reply:
Nice!